Apps/GatewayApi/test/unit/Controllers.Test/UserProfileControllerTests.cs

```diff
@@ -178,7 +178,7 @@ public async Task ShouldValidateAge()
                 new Mock<IUserSmsService>().Object,
                 new Mock<IAuthenticationDelegate>().Object);
 
-            PrimitiveRequestResult<bool> actualResult = await controller.Validate(this.hdid).ConfigureAwait(true);
+            var actualResult = await controller.Validate(this.hdid).ConfigureAwait(true);
```

Comment:
This use of var should be discussed with team. I think the use of var caused issues with warnings.  There was a PR in the past that cleaned these issues.  Please discuss with Team.... @MikeLyttle